packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -246,7 +246,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     return (
       <div {...divProps } ref='root' className={ this._classNames.container }>
         { label && (
-          <Label id={ id + '-label' } required={ required } htmlFor={ id } className={ this._classNames.label }>{ label }</Label>
+          <Label id={ id + '-label' } disabled={ disabled } required={ required } htmlFor={ id } className={ this._classNames.label }>{ label }</Label>
```

Comment:
Does this render correctly in high contrast?

Reply:
Let me check that 👍

Reply:
<img width="324" alt="screen shot 2017-10-11 at 5 04 00 pm" src="https://user-images.githubusercontent.com/3754893/31472901-3c5e0018-aea6-11e7-841a-19d706598b7c.png">

That is how it looks in high contrast mode... should I change the visual indication.. maye grey it out too for high contrast?

Reply:
That doesn't look good. The text should be rendering with greyText (if you are in high contrast black disabled text should be lime green)

Reply:
I updated the PR with a fix... (see the screenshot underneath)... does that look right?